hc/accounts/tests/test_check_token.py

```diff
@@ -22,7 +29,29 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_it_redirects_loggedin_users(self):
```

Comment:
Nice Implementation of the redirect after successful login, although having a doctoring would be best to explain the function

Reply:
Great! This will be implemented.